src/NuGet.Clients/PackageManagement.UI/Models/PackageStatus.cs

```diff
@@ -6,7 +6,10 @@ namespace NuGet.PackageManagement.UI
     public enum PackageStatus
     {
         NotInstalled,
+
+        // the latest version is installed
```

Comment:
Given pre-release flag and also After the change to respect version ranges in the packages.config, this should become the latest applicable version